pkg/standalone/publish_test.go

```diff
@@ -0,0 +1,97 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPubLish(t *testing.T) {
+	testCases := []struct {
+		name          string
+		pubsubName    string
+		payload       string
+		topic         string
+		lo            ListOutput
+		listErr       error
+		expectedPath  string
+		postResponse  string
+		resp          string
+		errorExpected bool
+		errString     string
+	}{
+		{
+			name:          "test empty topic",
+			payload:       "test",
+			pubsubName:    "test",
+			errString:     "topic is missing",
+			errorExpected: true,
+		},
+		{
+			name:          "test empty pubsubName",
+			payload:       "test",
+			topic:         "test",
+			errString:     "pubsubName is missing",
+			errorExpected: true,
+		},
+		{
+			name:          "test list error",
+			payload:       "test",
+			topic:         "test",
+			pubsubName:    "test",
+			listErr:       assert.AnError,
+			errString:     assert.AnError.Error(),
+			errorExpected: true,
+		},
+		{
+			name:       "test empty appID in list output",
+			payload:    "test",
+			topic:      "test",
+			pubsubName: "test",
+			lo: ListOutput{
+				// empty appID
+				Command: "test",
+			},
+			errString:     "couldn't find a running Dapr instance",
+			errorExpected: true,
+		},
+		{
+			name:         "successful call",
+			pubsubName:   "testPubsubName",
+			topic:        "testTopic",
+			payload:      "test payload",
+			expectedPath: "/v1.0/publish/testPubsubName/testTopic",
+			postResponse: "test payload",
+			lo: ListOutput{
+				AppID: "notempty",
+			},
+		},
+	}
+	for _, tc := range testCases {
+		tc := tc
```

Comment:
Same comment as the other instance.. this is not needed for sequential `t.Run(...)`

Reply:
Same.

Reply:
Ah, I was reviewing outdated PR. Sorry about that :)!